src/library_int53.js

```diff
@@ -57,7 +57,7 @@ mergeInto(LibraryManager.library, {
 
   // Uint64 variant of writeI53ToI64Clamped. Writes the Number to a Uint64 variable on
   // the heap, clamping out of range values to range [0, 2^64-1].
-  $writeI53ToU64Clamped: function(ptr, num) {
+  $writeI53ToU64Clamped: (ptr, num) => {
```

Comment:
Do these changes make a difference in the final wasm? I picked e.g. `emscripten_get_now` and switched it like this:
```diff
diff --git a/src/library.js b/src/library.js
index 31901ce36..bac2eea8a 100644
--- a/src/library.js
+++ b/src/library.js
@@ -2352,7 +2352,7 @@ mergeInto(LibraryManager.library, {
 #endif
 `,
 
-  emscripten_get_now_res: () => { // return resolution of get_now, in nanoseconds
+  emscripten_get_now_res: function() { // return resolution of get_now, in nanoseconds
 #if ENVIRONMENT_MAY_BE_NODE
     if (ENVIRONMENT_IS_NODE) {
       return 1; // nanoseconds
```
Building with `-O3 -g1` I don't see any output difference, the result is still
```js
var _emscripten_get_now;

_emscripten_get_now = () => performance.now();
```
I see the same with `-O1`. Does the JS emitter already standardize these?

Reply:
Oh, wait, never mind... I unluckily picked `emscripten_get_now` which is actually handled a special way anyhow. (Also I missed that I modified the `_res` function and not it itself..)